bootstrap/kubeadm/docs/external-etcd.md

```diff
@@ -1,49 +1,76 @@
+Cluster API Bootstrap Provider Kubeadm supports using an external etcd cluster for your workload Kubernetes clusters.
 
 ## ⚠️ Warnings ⚠️
 
 Before getting started you should be aware of the expectations that come with using an external etcd cluster.
 
 * Cluster API is unable to manage any aspect of the external etcd cluster.
 * Depending on how you configure your etcd nodes you may incur additional cloud costs in data transfer.
-    * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd
-    across availability zones, but if you do please be aware of the costs.
+    * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd across availability zones, but if you do please be aware of the costs.
 
 ## Getting started
 
-To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client key/pair.
-This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
+To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client certificate and private key. This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
  
 ### Setting up etcd with kubeadm 
 
-CA certificates are required to setup etcd cluster.
-If you already have a CA then the CA's `key` and `crt` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`. 
+CA certificates are required to setup etcd cluster. If you already have a CA then the CA's `crt` and `key` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`. 
 
-If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files
+If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files:
 
- * `/etc/kubernetes/pki/etcd/ca.crt`
- * `/etc/kubernetes/pki/etcd/ca.key`  
+* `/etc/kubernetes/pki/etcd/ca.crt`
+* `/etc/kubernetes/pki/etcd/ca.key`  
 
-These key/pair are used to sign etcd server, peer certificates and eventually apiserver-etcd client. More information on how to setup external etcd with kubeadm can be found [`here`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/#setting-up-the-cluster).
+This certificate and private key are used to sign etcd server, peer certificates and eventually apiserver-etcd-client certificate. More information on how to setup external etcd with kubeadm can be found [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/#setting-up-the-cluster).
```

Comment:
kubeadm also uses the etcd CA to sign `healthcheck-client.{crt|key}`, which are more appropriate to have been called `client...` as the resulting TLS configuration is used for more actions than just "healthcheck". the `healthcheck-` part is residue from the days when kubeadm used a bash/etcdctl liveness probe for etcd server instances.

leaving it to you if you want to document it as e.g. just `client certificates`, or omit this part.

Reply:
I've updated it in 1876bc044, PTAL.